Header header
float64[18] tip_pos
float64[18] tip_vel
float64[18] tip_fce
